Vagrantfile

```diff
@@ -137,8 +137,6 @@ kubernetes_minion_1_ipv6 = network_prefix_ipv6 + settings["net"]["minion_1_ipv6_
 kubernetes_minion_2_ipv6 = network_prefix_ipv6 + settings["net"]["minion_2_ipv6_part"]
```

Comment:
Reminder: Add an echo that shows the currently selected provider.

Reply:
done